records_mover/url/s3/s3_directory_url.py

```diff
@@ -35,11 +42,22 @@ def purge_directory(self) -> None:
             self.s3_resource.meta.client.delete_objects(Bucket=self.bucket, Delete=delete_keys)
 
     def copy_to(self, other_loc: BaseDirectoryUrl) -> BaseDirectoryUrl:
+        from ..gcs.gcs_directory_url import GCSDirectoryUrl
         if not other_loc.is_directory():
             raise RuntimeError(f"Cannot copy a directory to a file ({other_loc.url})")
         elif isinstance(other_loc, FilesystemDirectoryUrl):
+            # TODO: Should thi salso be part of optimizer?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker